From 99cb935fbf21590ddc4660639a308a96ee832f23 Mon Sep 17 00:00:00 2001
From: K900 <me@0upti.me>
Date: Thu, 27 Apr 2023 00:42:21 +0300
Subject: [PATCH] telegram-desktop: remove crimes

This may have worked 4 years ago, but it most definitely does not work on the binary wrapper.
---
 .../instant-messengers/telegram/telegram-desktop/default.nix | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/pkgs/applications/networking/instant-messengers/telegram/telegram-desktop/default.nix b/pkgs/applications/networking/instant-messengers/telegram/telegram-desktop/default.nix
index 33c943e3bc97f..254540bf8f93c 100644
--- a/pkgs/applications/networking/instant-messengers/telegram/telegram-desktop/default.nix
+++ b/pkgs/applications/networking/instant-messengers/telegram/telegram-desktop/default.nix
@@ -181,10 +181,7 @@ stdenv.mkDerivation rec {
       "''${gappsWrapperArgs[@]}" \
       "''${qtWrapperArgs[@]}" \
       --prefix LD_LIBRARY_PATH : "${xorg.libXcursor}/lib" \
-      --suffix PATH : ${lib.makeBinPath [ xdg-utils ]} \
-      --set XDG_RUNTIME_DIR "XDG-RUNTIME-DIR"
-    sed -i $out/bin/telegram-desktop \
-      -e "s,'XDG-RUNTIME-DIR',\"\''${XDG_RUNTIME_DIR:-/run/user/\$(id --user)}\","
+      --suffix PATH : ${lib.makeBinPath [ xdg-utils ]}
   '';
 
   passthru = {
